cmd/record.go

```diff
@@ -227,6 +233,9 @@ func (r *Record) GetCmd() *cobra.Command {
 
 	recordCmd.Flags().String("config-path", ".", "Path to the local directory where keploy configuration file is stored")
 
+	recordCmd.Flags().Bool("enableTele", true, "Switch for telemetry")
+	_ = recordCmd.Flags().MarkHidden("enableTele")
```

Comment:
I think this assignment is optional. Please remove assignment if it is optional.

Reply:
Done